addOns/client/src/main/java/org/zaproxy/addon/client/spider/ClientSpiderDialog.java

```diff
@@ -115,6 +128,16 @@ private void init(SiteNode node, String url) {
             this.addUrlSelectField(0, FIELD_START, url, true, false);
         }
 
+        this.addComboField(0, FIELD_CONTEXT, new String[] {}, "");
+        List<String> ctxNames = new ArrayList<>();
+        ctxNames.add("");
+        Session session = Model.getSingleton().getSession();
+        session.getContexts().forEach(context -> ctxNames.add(context.getName()));
+
+        this.setComboFields(FIELD_CONTEXT, ctxNames, "");
+        this.addComboField(0, FIELD_USER, new String[] {}, "");
+        this.addFieldListener(FIELD_CONTEXT, e -> setUsers());
```

Comment:
Would be better not show them if we can't use them because the extension is not enabled.

Reply:
`this.addComboField(0, FIELD_CONTEXT, new String[] {}, "");` should be wrapped too.

Reply:
OOps moved